integration/flink/app/build.gradle

```diff
@@ -0,0 +1,368 @@
+/*
+/* Copyright 2018-2023 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+import org.apache.tools.ant.filters.ReplaceTokens
+import io.franzbecker.gradle.lombok.task.DelombokTask
+
+plugins {
+    id 'java'
+    id 'java-library'
+    id 'maven-publish'
+    id 'signing'
+    id 'jacoco'
+    id "com.adarshr.test-logger" version "3.2.0"
+    id 'com.diffplug.spotless' version '6.25.0'
+    id "com.github.johnrengelman.shadow" version "8.1.1"
+    id "pmd"
+    id 'io.franzbecker.gradle-lombok' version '5.0.0'
+}
+
+pmd {
+    consoleOutput = true
+    toolVersion = "6.46.0"
+    rulesMinimumPriority = 5
+    ruleSetFiles = rootProject.files("pmd-openlineage.xml")
+    ruleSets = []
+    ignoreFailures = true
+}
+
+pmdMain {
+    dependsOn shadowJar
+    reports {
+        html.required = true
+    }
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven {
+        url = "https://packages.confluent.io/maven/"
+    }
+}
+
+configurations {
+    lombok
+}
```

Comment:
You declare this configuration, but I don't see it being used. Can you point me to the spot where you use this configuration?

Reply:
Removed